include/multiple_object_tracking/unscented_kalman_filter.hpp

```diff
@@ -74,8 +74,17 @@ inline auto unscented_kalman_filter_predict(
   }
 
   // Convert mean and sigma points into Eigen::MatrixXf
-  const auto m_sigma_points{
-    mean_and_sigma_pints_to_matrix_xf(predicted_mean, predicted_sigma_points)};
+  auto m_sigma_points{mean_and_sigma_pints_to_matrix_xf(predicted_mean, predicted_sigma_points)};
```

Comment:
mean_and_sigma_pints_to_matrix_xf

has a typo. Can we fix it while we are at it? (says pints instead of points). Will there be any case where the angle is more than -360 degrees?

Reply:
No because the vectors get generated from classes with auto-wrapping angles.